### PR TITLE
Add baseball recipe

### PR DESCRIPTION
**Pull request by juliavetter**
#### Summary
Balance "Add baseball recipe"

#### Purpose of change

In innawood, there was no way to use the dodging practice recipe, since you need some kind of ball. A baseball seems easy enough to craft that this could be added to mainline as well. 

#### Describe the solution

I added a recipe to craft a baseball from leather, strings, and glue. This can be used to practice dodging (or as a thrown weapon). 

#### Testing

Started the game, confirmed the baseball can be crafted with the specified components and then disassembled. 

#### Additional context

The recipe requires fabric cutting 1, which is not available innawood until you get metal working, which makes the recipe not so useful. But a pre-metal fabric cutting item may be added in the future.

### CODE REVIEW

**Review by Light-Wave — APPROVED** [review]

**Comment by kurtisvetter** [comment]
Test failures seem unrelated I think?

**Comment by kevingranade** [comment]
Just a FYI as I'm not the realism police for mods, you're pretty heavily under representing the amount of string in a baseball.
https://bluemonkeysports.com/how-much-string-is-in-a-baseball/
tl;dr 369 yards

**Comment by kurtisvetter** [comment]
>tl;dr 369 yards

!! I was just going off the existing disassembly recipe for baseballs. 369 yards is around 185 long strings/cordages. WIth cordages that would probably be doable, if wildly inefficient. But with strings, it would take a few weeks of constantly dissasembling fibrous stalks to get that much string. May have to rethink this one...

**Comment by Light-Wave** [comment]
I much appreciate your input, @kevingranade
A baseball in-game weight 0.33 lbs. A long string weight 0.08 lbs. 185 long strings would weight 14.8 lbs, or almost 45 baseballs. A baseball made entirely of long strings would be made out of 4.125 long strings. Perhaps the string used in real balls is thinner than what we are using in game?
2 to 3 long strings sounds like a reasonable amount in my opinion.

**Comment by kevingranade** [comment]
If I had to guess I'd say the string recovered in the disassembly recipe represents the stitching as that's heavy duty twine, where the filling is a lightweight yarn.  The person that made the recipe might have not known about the filling or discounted it as not useful and just had it recovering the covering.

My best guess at how much string to use for filling is to ignore the length and just look at weight, the string item we use is a relatively thick/heavy twine, but that means it's going to take up more space too.  It's not going to be an "official MDB basebal", but for the purposes of the game it doesn't matter.  With that adjustment you could feel free to push it to the main game if you like.

Potentially the disassembly recipe should yield a similarly matched weight of fiber instead of huge amounts of string, but that's very much a side issue.
